dist/openshift/cincinnati-deployment.yaml

```
@@ -298,6 +332,26 @@ objects:
         port = ${GB_STATUS_PORT}
 
         ${GB_PLUGIN_SETTINGS}
+
+  - apiVersion: keda.sh/v1alpha1
+    kind: ScaledObject
+    metadata:
+      name: cincinnati-pe-scaler
+      labels:
+        app: cincinnati-policy-engine
+    spec:
+      scaleTargetRef:
+        name: cincinnati-policy-engine
+      maxReplicaCount: ${{PE_MAX_REPLICAS}}
+      minReplicaCount: ${{PE_MIN_REPLICAS}}
+      triggers:
+        - type: prometheus
+          metadata:
+            serverAddress: http://prometheus-app-sre.openshift-customer-monitoring.svc.cluster.local:9090
+            metricName: cincinnati_policy_engine_graph_incoming_requests_rate
```

Reference of a removed metric as the PrometheusRule cincinnati-recording-rule is no longer present. I am not that knowledgeable regarding the APIs, feel free to correct me.

---

Shouldn't `cincinnati_pe_graph_incoming_requests_total` be used here as it is used in the query?

```suggestion
            metricName: cincinnati_pe_graph_incoming_requests_total
```